src/BitmapSkin.js

```diff
@@ -71,11 +78,9 @@ class BitmapSkin extends Skin {
             gl.texImage2D(gl.TEXTURE_2D, 0, gl.RGBA, gl.RGBA, gl.UNSIGNED_BYTE, bitmapData);
             this._silhouette.update(bitmapData);
         } else {
+            // TODO: mipmaps?
```

Comment:
A mipmapped texture occupies 4/3 as much memory as a non-mipmapped texture, but usually looks better and can improve performance on weaker graphics chips.

Reply:
This would be good a file a follow-up issue for rather than tracking in `@todo`s.